cli/types/cypress.d.ts

```diff
@@ -570,6 +570,14 @@ declare namespace Cypress {
      */
     log(options: Partial<LogConfig>): Log
 
+    /**
+     * Stop the Cypress App on the current machine while tests are running
+     * @see https://on.cypress.io/cypress-stop
```

Comment:
the link fails for me. Does an onlink PR need to be added?

Reply:
yah I'm gonna open a PR for this to our on service

Reply:
PR here: https://github.com/cypress-io/cypress-services/pull/10336